portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +31,57 @@ function addRandomFact() {
   const factContainer = document.getElementById('randomFact');
   factContainer.innerText = fact;
 }
+
+/**
+* Retrieves comments when page loads.
+*/
+const getComments = () => {
+  fetch('/comments').then(response => response.json()).then((comments) => {
+    const commentContainerElement = document.getElementById('comment-container');
+    const titleElement = document.createElement("h2");
+
+    //Clean element
+    commentContainerElement.innerHTML = '';
+
+    //If no comments, display an alert, else append comments to the container element.
+    if(comments.length === 0) {
+      titleElement.innerHTML = "There are no comments 😕, be the first!";
+      commentContainerElement.appendChild(titleElement);
+    }
+    else {
+      titleElement.innerHTML = "Last comments";
+      commentContainerElement.appendChild(titleElement);
+      appendComments(comments, commentContainerElement);
+    }
+  });
+};
+
+/**
+* Creates HTML elements for each comment
+* And appends them into the father element
```

Comment:
Please include JSDoc for this function (@param {!Array<Object>}, @param {element} ) ;you can do this in a follow-up if you want to :)